test/ctia/entity/event_test.clj

```diff
@@ -69,7 +69,11 @@
            (fixture-with-fixed-time
             (time/timestamp "2042-01-01")
             (fn []
-              (let [{incident :parsed-body
+              (let [app (helpers/get-current-app)
+                    {:keys [get-in-config]} (helpers/get-service-map app :ConfigService)
+
+                    port (get-in-config [:ctia :http :port])
```

Comment:
These tests failed on my machine because the port was hardcoded in the data below. Factored out the port to be a function of the current config.